tark/tark_web/templates/mane_GRCh37_list.html

```diff
@@ -1,130 +1,123 @@
 {% extends "web_home.html" %}
-
 {% load staticfiles %}
 {% load add_class %}
-{% load search_result_formatter %}
 
+{% load search_result_formatter %}
 {% block content %}
-
 <div class="container" style="padding-top:20px;">
     
     <!-- Info -->
     <div class="panel panel-success">
         
-        <div class="panel-heading"><i class="glyphicon glyphicon-info-sign"></i> Table of matches from GRCh37 showing equivalence in 5' UTR, 3' UTR and CDS in relation to 
-            <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2018/10/11/matched-annotation-by-ncbi-and-embl-ebi-mane-a-new-joint-venture-to-define-a-set-of-representative-transcripts-for-human-protein-coding-genes/" target="_blank">MANE</a> 
-            Select and MANE Plus Clinical transcripts on GRCh38.</div> 
+        <div class="panel-heading"><i class="glyphicon glyphicon-info-sign"></i> Table of matches from GRCh37 showing equivalence  in 5' UTR, 3' UTR and CDS in relation to MANE Select and MANE Plus Clinical transcripts on GRCh38</div>
+        <div class="panel-body">
+            
+            <table id="feature_table" class="table table-striped table-bordered table-responsive table-condensed text-center">
+                <thead>
+                    <tr>
+                        <th>Gene</th>
+                        <th>MANE TYPE</th>
+                        <th>Ensembl StableID GRCh38</th>
+                        <th>RefSeq StableID <br>GRCh38 / GRCh37</th>
+                        <th>Ensembl StableID GRCh37<br> (Not MANE)</th>
+                        <th>5'UTR</th>
+                        <th>CDS</th>
+                        <th>3'UTR</th>
+                    </tr>
+                </thead>
+            </table>
+        </div>
+    </div>
+</div>
+
+<!-- Refer content under 2nd heading in https://www.ebi.ac.uk/seqdb/confluence/display/EA/One+time+scripts+related+to+Mane+-+GRCh37+page -->
+
+<script>
+    $(document).ready(function() {
+        $('[data-toggle="tooltip"]').tooltip();
+        var requestUrl = "https://raw.githubusercontent.com/Ensembl/tark-loader/dummy_ea818/mane_grch37.txt";
```

Comment:
@ranjit-s Maybe copying the file to project space on NFS and reading it from there is the better way to do this. As discussed in the meeting today, we could have done this later but we will then risk not implementing it.

Reply:
Just to make sure I understood this right, this is the same file created using `utr_cds_to_json.py` script right?
Because that script creates `mane_grch37.txt` file under `[...]/tark/tark/tark/static/csv/mane_grch37.txt`, can that location be used here?

Reply:
Hi @sanjay-boddu  and @bilalebi 
Thanks a lot for your review comments!

a. the mane_GRCh37_list/  has been modified with the URL now project space instead of GitHub repo link
b. Please find the changes incorporated into the one-time script (provided in confluence(https://www.ebi.ac.uk/seqdb/confluence/display/EA/One+time+scripts+related+to+Mane+-+GRCh37+page):
> included the comment above `import mysql.connector` mentioning that MySQL connector is required (pip install mysql-connector-python)
> replaced ensrw with `ensro` , and removed password placeholder as it is no longer necessary
> the sql query has been put inside a string. 
> the .txt file will be downloaded to /homes/XXXtarkdevXXX/.../tark/static/txt/ and /homes/XXXtarkdevXXX/..../tark/static/apache/txt/

c. At the moment I do not have permissions to create a repository under Ensembl. I can upload the scripts there if one of you can create a repo and provide me write access.